docs/src/main/tut/datatypes/repl.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: docs
+title:  "Learning Algebird at the REPL"
+section: "data"
+---
+
+# Learning Algebird Monoids at the REPL
```

Comment:
Be good to link to this from the parent page, shows up in the index, but be good to be easier to find